addons/website_sale/controllers/delivery.py

```diff
@@ -13,51 +13,50 @@
 class WebsiteSaleDelivery(WebsiteSale):
     _express_checkout_shipping_route = '/shop/express/shipping_address_change'
 
-    @route('/shop/delivery_carriers', type='json', auth='public', website=True, csrf=False)
-    def shop_delivery_carriers(self):
-        """ TODO ANVCHU """
+    @route('/shop/delivery_methods', type='json', auth='public', website=True, csrf=False)
+    def shop_delivery_methods(self):
+        """ Fetch available delivery methods and render them in the delivery form. """
         order = request.website.sale_get_order()
-        values = {
-            'order': order,
-            'delivery_methods': order._get_delivery_methods(),
-        }
+        values = {'order': order, 'delivery_methods': order._get_delivery_methods()}
         return request.env['ir.ui.view']._render_template("website_sale.delivery_form", values)
 
-    @route('/shop/update_carrier', type='json', auth='public', methods=['POST'], website=True)
-    def shop_update_carrier(self, carrier_id=None, **kwargs):
-        """ TODO ANVCHU """
+    @route(
+        '/shop/set_delivery_method', type='json', auth='public', methods=['POST'], website=True
+    )
+    def shop_set_delivery_method(self, dm_id=None, **kwargs):
+        """ Update order delivery method if it was changed and return order summary values. """
         order = request.website.sale_get_order()
```

Comment:
```suggestion
        order_sudo = request.website.sale_get_order()
```
while you modify those methods